src/Stratis.Bitcoin/P2P/PeerConnector.cs

```diff
@@ -260,7 +261,7 @@ internal async Task ConnectAsync(PeerAddress peerAddress)
             }
             catch (Exception exception)
             {
-                this.logger.LogTrace("Exception occurred while connecting: {0}", exception.ToString());
+                this.logger.LogTrace("Exception occurred while connecting: {0}", exception is SocketException ? exception.Message : exception.ToString());
```

Comment:
I thin this is the only change needed in the entire PR (the excessive trace of handled errors)

Reply:
Just add another catch block and catch type is SocketException then trace the message.